src/main/java/com/aws/greengrass/deployment/DefaultDeploymentTask.java

```diff
@@ -163,23 +165,18 @@ public DeploymentResult call() throws InterruptedException {
             //   on the device before the deployment started, and finding one of type nucleus.
             Optional<DeploymentPackageConfiguration> incomingNucleusComponentConfiguration =
                     deploymentDocument.getDeploymentPackageConfigurationList() == null ? Optional.empty() :
-                    deploymentDocument.getDeploymentPackageConfigurationList().stream()
-                        .filter(c -> c.getPackageName().equals(deviceConfiguration.getNucleusComponentName()))
-                        .findAny();
+                            deploymentDocument.getDeploymentPackageConfigurationList().stream()
```

Comment:
nit: can you revert this formatting change?

Reply:
I agree with @muboheng, the new indentation does make sense.

Over-arching comment - do we not have formatters configured which can take care of these for us?